model_evaluation/utils_export/dataset_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+
+import pandas as pd
+
+from utils_export.dataset import Dataset, Model
+from utils_export.utils_cloudml import FeatureSpec
+
+
+class TestCompatibleInputFn(unittest.TestCase):
+  """Tests to verify that the compatibility between input_fn and model."""
```

Comment:
Remove 'that' (i.e. "verify the compatibility")

Reply:
Also, this test dosn't involve the model. It just tests if the input function is valid. Please fix description.